tests/baselines/reference/asOpEmitParens.js

```diff
@@ -0,0 +1,19 @@
+//// [asOpEmitParens.ts]
+declare var x;
+// Must emit as (x + 1) * 3
+(x + 1 as number) * 3;
```

Comment:
You're looking at the wrong part of the baseline (this is the original .ts). Its corresponding emitted line is 15